gptqmodel/models/base.py

```diff
@@ -137,7 +137,7 @@ def _convert_tensor_to_list(tensor):
             pad_token_list = ["<pad>", "<|pad|>", "<|finetune_right_pad_id|>"]
 
             tokenizer = AutoTokenizer.from_pretrained(self.model.name_or_path)
```

Comment:
tokenizer should not be create here..should be passed in to method from parent caller